examples/ansible/setup_libvirt_host.yml

```diff
@@ -4,10 +4,14 @@
 # A example playbook showing how the setup an ALP system as a libvirt host.
 #
 
+# use ansible-playbook setup_libvirt_host.yml -e "playbook_path=$(pwd)" -e "ansible_user=$(whoami)" -e "reboot_mode=auto" to enable auto reboot while running as root ensure ansible container is installed as root.
 
 - name: Setup ALP system as a libvirt host
   hosts: alphost
   vars:
+    ansible_playbook_basename: "setup_libvirt_host.yml"
+    ansible_user: "{{ ansible_user }}"
```

Comment:
This defined ansible_user as itself, which should be a value that originally comes from the inventory specifying what alphost is.

I'm not sure you should be doing this is the correct thing to do here, or makes sense.